src/App.js

```diff
@@ -62,9 +62,7 @@ class App extends Component {
             </div>
           </Router>
         ) : (
-          <div>
-            <FormattedMessage id="loading" />
-          </div>
+          <Loading />
```

Comment:
Great! I think this will be used a lot. And it will made easier to refactor into a spinner or whatever later on.